### DISCUSSION TITLE
Simulation courant number shoots up even with very low delta t

### DISCUSSION THREAD

**Discussion by Platinumd1991** [Nek5000/nekRS#480] [General]
I have been using NekRS very successfully for multiple high Re flows. However, recently I have tried to run Re = 2240 for a new project and the courant number keeps shooting up regardless if I lowered delta t or refined the mesh. I ran the same case with Nek5000 and no issues like this occur. Is this a bug with NekRS?

**step= 52598 t= 1.82207840e+02 dt=8.0e-05 C= 4.27 UVW: 2 P: 22 eTime=
> > > 2.11e-01, 2.71779e+03 s
> > > step= 52599 t= 1.82207920e+02 dt=8.0e-05 C= 5.82 UVW: 3 P: 23 eTime=
> > > 2.09e-01, 2.71800e+03 s
> > > step= 52600 t= 1.82208000e+02 dt=8.0e-05 C= 8.76 UVW: 3 P: 25 eTime=
> > > 2.30e-01, 2.71823e+03 s
> > > copying solution to nek
> > > calling nek_userchk ...
> > > 52600 1.82208000000E+02 7.26226313888E+00 7.26559569350E+00
> > > -3.33255462393E-03 1dragx
> > > 52600 1.82208000000E+02 -2.88376652997E-03 -2.62383438671E-03
> > > -2.59932143255E-04 1dragy
> > > 52600 1.82208000000E+02 -2.16552739071E-04 1.40683273030E-16
> > > -2.16552739071E-04 1dragz
> > > 52600 1.82208000000E+02 7.26226313888E+00 7.26559569350E+00
> > > -3.33255462393E-03 2dragx
> > > 52600 1.82208000000E+02 -2.88376652997E-03 -2.62383438671E-03
> > > -2.59932143255E-04 2dragy
> > > 52600 1.82208000000E+02 -2.16552739071E-04 1.40683273030E-16
> > > -2.16552739071E-04 2dragz
> > > dump history points
> > > step= 52601 t= 1.82208080e+02 dt=8.0e-05 C= 15.38 UVW: 3 P: 30 eTime=
> > > 3.06e-01, 2.71854e+03 s
> > > step= 52602 t= 1.82208160e+02 dt=8.0e-05 C= 45.05 UVW: 3 P: 37 eTime=
> > > 3.26e-01, 2.71886e+03 s
> > > Unreasonable CFL! Dying ...
> > >**

**Comment by stgeke** [comment]
Could you please provide more details about your case.

**Comment by Platinumd1991** [comment]
Hi,

Apologies for the late reply. I am running a 3D flat plate at Re = 2240 in
a confined setting( no-slip around walls). Please find a sample of my case.

 compile_nekRS.slurm
<https://drive.google.com/file/d/1l698rqMkIwYoLob0e8sJJkMSeJfoiFAH/view?usp=drive_web>
 fpre2240_B00_M1_3D.oudf
<https://drive.google.com/file/d/1GzRd0cZWiDV8_OLJ5_ZlyoEfcYbbwIE0/view?usp=drive_web>
 fpre2240_B00_M1_3D.par
<https://drive.google.com/file/d/1P2fZNdEITFZmCivSHQazbd_V0HFNI9jj/view?usp=drive_web>
 fpre2240_B00_M1_3D.re2
<https://drive.google.com/file/d/1JcKIFqDXp5e07owGiR82iLsru0j78wsj/view?usp=drive_web>
 fpre2240_B00_M1_3D.slurm
<https://drive.google.com/file/d/1bAezztXwy6WN0T0HxBi93IQviOSMyCk9/view?usp=drive_web>
 fpre2240_B00_M1_3D.udf
<https://drive.google.com/file/d/1oayhZ-Dq9bsYdhLY_sel7DEsK9NnpYGM/view?usp=drive_web>
 fpre2240_B00_M1_3D.usr
<https://drive.google.com/file/d/1iCu9k2RYlO_MErDaTzmpKHoMLJO51JPF/view?usp=drive_web>
